src/components/DittoComponent.tsx

```diff
@@ -3,17 +3,21 @@ import { useDittoComponent } from "../hooks/useDittoComponent";
 import { DittoComponentLibraryProps } from "./Ditto";
 
 export const DittoComponent = (props: DittoComponentLibraryProps) => {
-  const { children, componentId } = props;
+  const { children, componentId, variables = {}, count } = props;
+
   const value = useDittoComponent({
     componentId,
     alwaysReturnString: typeof children !== "function",
+    variables,
```

Comment:
```suggestion
    variables: variables || {},
```

For reasons described [here](https://github.com/dittowords/ditto-react/pull/10/files#r784865890)

Reply:
👍